app/_src/gateway/kong-manager/auth/oidc/configure.md

```diff
@@ -28,19 +28,21 @@ admin_gui_auth_conf={                                      \
   "issuer": "https://accounts.google.com/",                \
   "client_id": ["<ENTER_YOUR_CLIENT_ID>"],                 \
   "client_secret": ["<ENTER_YOUR_CLIENT_SECRET_HERE>"],    \
-  "consumer_by": ["username","custom_id"],                 \
-  "ssl_verify": false,                                     \
-  "admin_claim": ["email"],                             \
-  "leeway": 60,                                            \
-  "redirect_uri": ["http://localhost:8002/default"],                      \
-  "login_redirect_uri": ["http://localhost:8002/default"],                \
-  "logout_methods": ["GET", "DELETE"],                     \
-  "logout_query_arg": "logout",                            \
-  "logout_redirect_uri": ["http://localhost:8002/default"],               \
-  "scopes": ["openid","profile","email","offline_access"], \
-  "auth_methods": ["authorization_code"]                   \
+  "admin_claim": ["email"],                                \
+  "redirect_uri": ["http://localhost:8002/default"],       \
 }
 ```
+Required for `admin_gui_auth_conf` of `openid-connect`
```

Comment:
```suggestion
While authenticating Kong Manager with OpenID Connect, please ensure that your IdP supports the `authorization_code` grant type and is enabled for the associated client.

Configuration parameters in `admin_gui_auth_conf` for `openid-connect`:

```

### Add a note

To let users know that OpenID Connect with Kong Manager only supports the `authorization_code` grant type.

### Fix the table

<img width="1670" alt="Screenshot 2023-10-25 at 14 24 46" src="https://github.com/Kong/docs.konghq.com/assets/5277268/d6fcf452-0556-48b3-aabe-bd2d021a626a">

The table looks broken. We need to leave a blank line between the paragraph and the table to let the table render normally.

<img width="1418" alt="Screenshot 2023-10-25 at 14 28 25" src="https://github.com/Kong/docs.konghq.com/assets/5277268/da157391-2de9-4109-8f9f-12322d11b245">